code-reviews/recipes/Terraform.md

```diff
@@ -53,10 +58,16 @@ echo -e "\n\n>>> Terraform validate"
 terraform validate
 ```
 
-> TODO: Add example of how to use this in an AzDO build validation pipeline
-
-## Code Review Checklist
-
+### Code Review Checklist
 In addition to the [Code Review Checklist](../readme.md) you should also look for these terraform specific code review items
```

Comment:
General: seems there are a lot of setup recommendations rather than what to look for in a .tf file. Is there more to that we could add?

Reply:
@debugerr - Agreed, i feel what gets into a .tf is more of a general script best practice, but let me check with other experts and online and see if we can add more around this point.

Reply:
@debugerr - Added some critical items to look in the .tf file.

Reply:
Thanks!